bliss/utils.py

```diff
@@ -139,3 +140,42 @@ def forward(self, mean_z, logscale_z):
         if self.minscale is not None:
             logscale_z = torch.log(self.minscale + (1 - self.minscale) * F.softplus(logscale_z))
         return Normal(mean_z, logscale_z.exp())
+
+
+# adapt from torchvision.utils.make_grid
+@torch.no_grad()
```

Comment:
Does this basically wrap the function call in a `with torch.no_grad` block?

Reply:
I believe so; this is only ever used for plotting and might be memory-intensive with gradient tracking